packages/sanity/src/_internal/cli/actions/build/buildAction.ts

```diff
@@ -122,19 +117,29 @@ export default async function buildSanityStudio(
 
   const trace = telemetry.trace(BuildTrace)
   trace.start()
-  try {
-    timer.start('bundleStudio')
 
-    const vendorImports = enableAutoUpdates
-      ? await buildVendorDependencies({cwd: workDir, outputDir})
-      : {}
+  let importMap
 
-    const importMap = {
+  if (enableAutoUpdates) {
+    const version = encodeURIComponent(`^${installedSanityVersion}`)
+    const autoUpdatesImports = {
+      // /:packageName/:channel/:minVersion/:file?
+      'sanity': `https://api.sanity.work/v1/sanity/default/${version}`,
```

Comment:
The URL is incorrect for all, missing `modules` 

```suggestion
      'sanity': `https://api.sanity.work/v1/modules/sanity/default/${version}`,
```